node/src/main/scala/coop/rchain/node/effects/package.scala

```diff
@@ -67,10 +66,9 @@ package object effects {
   def rpConnections[F[_]: Concurrent]: F[ConnectionsCell[F]] =
     Ref[F].of(Connections.empty)
 
-  def rpConfState[F[_]: Monad: Sync](conf: RPConf): MonadState[F, RPConf] =
-    new AtomicMonadState[F, RPConf](AtomicAny(conf))
+  def rpConfState[F[_]: Sync](conf: RPConf): Ref[F, RPConf] = Ref.unsafe[F, RPConf](conf)
```

Comment:
Here it will not create difference in behavior, but better to use `Ref.of` when possible.

Reply:
[Done](https://github.com/rchain/rchain/pull/3778/commits/99174a0bec7f276f800e1ccb0d45cde7f5936537).